custom_components/tuya_local/devices/royalclima_fresh_climate.yaml

```diff
@@ -370,4 +370,20 @@ secondary_entities:
     dps:
       - id: 103
         type: boolean
+  - entity: sensor
+    name: Power
+    class: power
+    category: diagnostic
+    dps:
+      - id: 116
+        type: integer
+        name: sensor
+        precision: 0
```

Comment:
precision should not be needed, or at least allow 1 decimal place, as a divisor of 30 should give an accurate first decimal place.

Reply:
No, it results in a number with many digits after the decimal point. (The sensor itself is not very accurate, so I decided to show only int)